src/work/test/WorkTests.cpp

```diff
@@ -770,10 +770,10 @@ TEST_CASE("WorkSequence test", "[work]")
 class TestBatchWork : public BatchWork
 {
     bool mShouldFail;
-    int mTotalWorks;
+    size_t mTotalWorks;
 
   public:
-    int mCount{0};
+    size_t mCount{0};
```

Comment:
what was the warning here? Looks like we're using ints across the board from what I see

Reply:
`mTotalWorks` getting initialized with a `size_t`